src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  padding: 50px 40px;
+  width: 1024px;
```

Comment:
checklist 11th item, it should be an adaptive markup, use formula to calculate the max width for your container
```suggestion
```